solidity/random-beacon/test/RandomBeacon.Relay.test.ts

```diff
@@ -878,33 +973,60 @@ describe("RandomBeacon - Relay", () => {
           expect(await isGroupTerminated(0)).to.be.equal(true)
         })
 
-        it("should call staking contract to seize the min stake", async () => {
-          await expect(tx)
-            .to.emit(staking, "Seized")
+        it("should reward the notifier", async () => {
+          await expect(reportTx)
+            .to.emit(staking, "NotifierRewarded")
             .withArgs(
-              to1e18(100000),
-              params.unauthorizedSigningNotificationRewardMultiplier,
               notifier.address,
-              membersAddresses
+              constants.tokenStakingNotificationReward
+                .mul(params.unauthorizedSigningNotificationRewardMultiplier)
+                .div(100)
+                .mul(membersIDs.length)
             )
         })
 
+        it("should slash the minimum stake for all group members", async () => {
```

Comment:
It's not slashing the minimum stake but the slashing amount for unauthorized signing, right?

Reply:
f931643f8c9330c677ed937b457aed7e45072fe9